.github/workflows/build-publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and publish to ghcr.io
+
+on:
+  push:
+    branches: [ master ]
+
+env:
+  IMAGE_NAME: sonarcloud-github-action
+
+jobs:
+  build-publish:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Build image
+        run: docker build . --file Dockerfile --tag ghcr.io/$GITHUB_ACTOR/$IMAGE_NAME:`basename $GITHUB_REF` --label "runnumber=${GITHUB_RUN_ID}"
```

Comment:
I have a question here! In this case, any new tag will replace the old one on the registry side?
If yes, we don't need versioning tags for every new release?

Reply:
Yes, exactly. This workflow runs only on commits to master. Which in this project case are quite synchronized with releases 😄 
Joke aside, a workflow could be added to create a tagged image for each release tag. You would simply publish a release, and and image with the same tag would be built. If thats OK with you, I can add that to the PR.